src/app/[lng]/contact/page.tsx

```diff
@@ -1,5 +1,115 @@
-import React from 'react';
+"use client"
 
-const Datasets: React.FC = () => <div>page</div>;
+import { TextField, TextAreaField, Flex, Button, SelectField, Heading } from '@aws-amplify/ui-react';
+import React, { useRef, FormEvent } from 'react';
+import styled from 'styled-components';
 
-export default Datasets;
+const ContactForm = () => {
+
+
+    const firstNameRef = useRef<HTMLInputElement>(null);
+    const lastNameRef = useRef<HTMLInputElement>(null);
+    const emailRef = useRef<HTMLInputElement>(null);
+    const phoneNumberRef = useRef<HTMLInputElement>(null); 
+    const topicRef = useRef<HTMLSelectElement>(null);
+    const messageRef = useRef<HTMLTextAreaElement>(null);
+
+    const handleSubmit = async (e: FormEvent) => {
+        e.preventDefault();
+
+
+        const formData = {
```

Comment:
I'm curious as to why refs were required...I've never had to use that before to keep text fields in focus